src/octoprint/util/gcodeInterpreter.py

```diff
@@ -271,6 +280,8 @@ def _load(self, gcodeFile, printer_profile, throttle=None):
 	def _parseCuraProfileString(self, comment, prefix):
 		return {key: value for (key, value) in map(lambda x: x.split("=", 1), zlib.decompress(base64.b64decode(comment[len(prefix):])).split("\b"))}
 
+	def getWarning(self):
```

Comment:
I think that probably was rather meant for the "this is a bit backwards" comment further up? But yes, you are right.. I'll add a comment up there.